Foundation/FileHandle.swift

```diff
@@ -126,7 +126,9 @@ open class FileHandle : NSObject, NSSecureCoding {
         guard _fd >= 0 else { return }
         data.enumerateBytes() { (bytes, range, stop) in
```

Comment:
Just mentioning, looks like the method `enumerateBytes()` is deprecated in Swift 5.0. I am unsure if this PR is an appropriate place to make the changes, if not, I will update that the code in separate PR.

Reply:
I was trying to keep the fix as minimal as possible, I may update to it in a separate PR.